src/commands/workflow/run.ts

```diff
@@ -0,0 +1,47 @@
+import { Flags } from "@oclif/core";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { ApiError } from "@/lib/helpers/error";
+import { commaSeparatedArray, jsonStr } from "@/lib/helpers/flag";
+import {
+  formatErrorRespMessage,
+  isSuccessResp,
+  withSpinner,
+} from "@/lib/helpers/request";
+
+export default class WorkflowRun extends BaseCommand {
+  static flags = {
```

Comment:
Just merged #156 that fills out all the command --help summary docs, and I think it'd be nice to start filling them as we go now.